### PR TITLE
add audio export with linking in md file

### PR DESCRIPTION
**Pull request by benjaffe**
image
image
image

------------
image
image
image

---------
image
image
image
--------

@maxkrieger Please do a legit review on this, cause as of now, it's literally the only iOS code I've ever written. Tested in the simulator (screenshots above), but not tested on a real device.

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -59,6 +59,7 @@ Future<void> main() async {
   }
   void appRunner() => runApp(MultiProvider(
           providers: [
+            Provider<PlayerModel>(create: (_) => PlayerModel()..load()),
```
Comment (benjaffe) [inline lib/main.dart:62]: This bit in particular... I'm not sure if this is the right way to get the data in there.

**Comment by benjaffe** [comment]
This addresses https://github.com/maxkrieger/voiceliner/issues/55

File: lib/main.dart (hunk 1 above)
Comment (maxkrieger) [reply lib/main.dart:62]: Pretty sure this is redundant with the line right below?

**Comment by maxkrieger** [comment]
Will take a look in the coming days, thanks!

File: lib/main.dart (hunk 1 above)
Comment (benjaffe) [reply lib/main.dart]: Oh it does look that way! Just removed it.

**Comment by maxkrieger** [comment]
been a little busy - gimme another few!

**Comment by maxkrieger** [comment]
Rolling out soon, thanks ❤️ you're awesome dude

**Pull request merged**